docs/to_clarify.md

```diff
@@ -0,0 +1,8 @@
+## Question
+1. What are the "existing solutions and databases" that we need to integrate with as mentioned in Point 2 of "Possible Use Cases"?
+
+1. As mentioned under "Context" that "hotels... have tried different digital check-in solutions to limited success", are you able to show us exactly what are some of the solutions that have been tried and in which aspects are they lacking? Having these references would help us to come up with a better solution.
+
+1. Are there systems depending on the existing check-in/out system? If so, what are the API protocols used to communicate between these systems?
```

Comment:
What's an API protocol? Can't find such a term on google, and I can only find "API vs protocol" which indicates that these terms aren't meant to be used together.